src/libraries/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -850,6 +849,13 @@ private void DualModeConnect_BeginAccept_Helper(IPAddress listenOn, IPAddress co
                 serverSocket.Listen(1);
                 IAsyncResult async = serverSocket.BeginAccept(null, null);
                 SocketClient client = new SocketClient(_log, serverSocket, connectTo, port);
+                
+                Assert.True(
+                    client.WaitHandle.WaitOne(TestSettings.PassingTestTimeout),
+                    "Timed out while waiting for connection");
+                Assert.True(
+                        async.AsyncWaitHandle.WaitOne(TestSettings.PassingTestTimeout),
+                        "Timed out while to accept the client");
```

Comment:
```suggestion
                    async.AsyncWaitHandle.WaitOne(TestSettings.PassingTestTimeout),
                    "Timed out while waiting to accept the client");
```

Nit: indentation on the second assert